community/contribute/scholarships.md

```diff
@@ -0,0 +1,13 @@
+# Scholarships
+
+The [PEP-31 Scholarship Fund](https://forum.pokt.network/t/pep-31-scholarship-fund/) was [approved](https://gov.pokt.network/#/proposal/0x2920a4057b4aa6af033654bfcfdd77f2b98c81f100d8e55b97a30101e563db46) by the DAO in 06/2022 to allocate an accumulating monthly allowance of `10,000/POKT` per month to contributors, students, learners, speakers, educators and others or in the Pocket community.
+
+## Recipients
+
+### June 2022
+
+Tyler Kell ([@relyt29](https://twitter.com/relyt29)) is the first recipient of the Pocket Scholarship for his presentation `Academic Cryptocurrency Research Space - An Overview, Spring 2022` at [Infracon](https://www.infracon.org/) 2022 - Pocket's inaugral conference.
+
+The slides are available online [here](https://docs.google.com/presentation/d/1fJiSESRRC7qCpH0lZ9BlEynIV_AyBG8iQOucvnqlCNQ/edit?usp=sharing) and a PDF export is available in the repo [here](community/contribute/relyt29_06_2022.pdf).
+”
+A sum of 10,000 POKT was awarded to `bf50635d0a14a7faa6cf62494219be942cb9e21f` for this work!
```

Comment:
I think mentioning the address here is a nice touch, but could we go any further and link to the actual transaction? Otherwise, it's kind of just saying "look, here's Tyler's wallet". 🤷

Reply:
I don't think @jacklaing has sent it yet.

Reply:
I say we take out that line then. I don't think it adds much of anything, since we've already stated above that the recipient gets 10k POKT.

Reply:
Discussed offline. Going to wait for the transaction to come through for record-keeping purposes.

Reply:
Updated